client/protocol.py

```diff
@@ -48,12 +62,13 @@ async def build_from_addrs(cls, addrs: list[str]) -> ProtocolClient:
 
         connects = {}
         for member in cluster.members:
-            channel = grpc.aio.insecure_channel(member.name)
+            channel = grpc.aio.insecure_channel(member.addrs[0])
```

Comment:
The definition of grpc connect in the original design was like `connects: dict[server_id, grpc.Channel]`,  i.e. a server id corresponds to a grpc connect. In the next refactoring it will become like `connects: dict[server_id, List[grpc.Channel]]`, i.e a server id corresponds to grpc connects with multiple addresses.

Reply:
You are right. We shoud raise an exception when the addrs is empty.